src/components/SortableComponent.jsx

```diff
@@ -5,15 +5,16 @@ import 'react-sortable-tree/style.css';
 
 const SortableComponent = (props) => {
   const rootComponents = props.components.filter(
-    comp => comp.parentId.length === 0,
+    comp => comp.parentIds.length === 0,
   ).reverse();
+
   return (
     <div className="sortable-tree">
       <SortableTree
         style={{ backgroundColor: 'rgb(37, 37, 38)' }}
         treeData={rootComponents}
         canDrag={false}
-        onChange={treeData => this.setState({ treeData })}
+        onChange={() => {}}
```

Comment:
An empty function?

Reply:
The onChange function is required, and we are currently not using it.